packages/rOpenScPCA/tests/testthat/test-evaluate-clusters.R

```diff
@@ -0,0 +1,40 @@
+suppressPackageStartupMessages(library(SingleCellExperiment))
+
+set.seed(2024)
+sce <- splatter::simpleSimulate(nGenes = 1000, verbose = FALSE) |>
+  scater::logNormCounts() |>
+  scater::runPCA(ncomponents = 10)
+test_mat <- reducedDim(sce, "PCA")
+
+
+cluster_df <- calculate_clusters(test_mat)
+cluster_df$random_extra_column <- 10
+
+
+test_that("calculate_silhouette works as expected", {
+  df <- calculate_silhouette(test_mat, cluster_df)
+
+  expect_setequal(
+    colnames(df),
+    c("cell_id", "cluster", "algorithm", "weighting", "nn", "resolution", "silhouette_width", "other", "random_extra_column")
```

Comment:
Maybe make this a bit more compact (and not affected by changes to `calculate_clusters()`
```suggestion
    c(colnames(cluster_df), "silhouette_width", "other")
```

(Same suggestion below)